src/blocks/product-card.css

```diff
@@ -0,0 +1,109 @@
+/* #region Product Card */
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  padding: 0 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  align-self: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px 0;
+}
+
+.product-card__name {
+  margin: 8px 0;
+}
```

Comment:
Set margins to one side

Reply:
<img width="653" alt="Screenshot 2022-08-19 at 15 15 13" src="https://user-images.githubusercontent.com/84275496/185616370-9599d2a9-da41-4d2c-a18b-8df654c6b25c.png">